t/perf/benchmarks

```diff
@@ -2185,6 +2185,12 @@
         code    => 'for my $x (@a) {next}',
     },
 
+    'loop::for::redo4' => {
+        desc    => 'loop hitting redo a bunch of times',
+        setup   => 'my ($y, $x);',
+        code    => '$y = 0; $x = 2; out: $x--; while ($x) { goto out if $y++ > 4; redo; }',
+    },
```

Comment:
This benchmark seems overly complex. How about
'pre' => '$x =4',
code => 'for (1) { redo if --$x }'